### PR TITLE
LOG-1331: Setup CI for logging 5.1 release

### PR DESCRIPTION
**Pull request by jcantrill**
This PR

* sets up CI for logging 5.1 release
* updates master to 5.2

cc @vimalk78 @ewolinetz

### CODE REVIEW

**Review by alanconway — APPROVED** [review]
/approve
/lgtm
It seems to me that we should be able to centralize the version better, it's a lot of trivial manual changes.

**Comment by ewolinetz** [comment]
@jcantrill i think you missed regenerating the config metadata for these job files

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -1844,6 +1844,12 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/**/eventrouter/openshift*-eventrouter-release-5.*.yaml:
```
Comment (ewolinetz) [inline core-services/prow/02_config/_plugins.yaml:1847]: probably want to revert these changes...

**Comment by jcantrill** [comment]
/assign @alvaroaleman

Dont expact the rehearsals to pass until the dependent PRs merge but everything else looks correct

**Review by droslean — APPROVED** [review]
